control/controlhttp/client.go

```diff
@@ -382,18 +383,27 @@ func (a *Dialer) dialURL(ctx context.Context, u *url.URL, addr netip.Addr) (*Cli
 func (a *Dialer) tryURLUpgrade(ctx context.Context, u *url.URL, addr netip.Addr, init []byte) (net.Conn, error) {
 	var dns *dnscache.Resolver
 
+	// Maintain prior behaviour where passing no logger to
```

Comment:
Maintain prior behavior from when?

Reply:
Changed the approach here to pass along `a.Logf` directly